spec/dataframe_spec.rb

```diff
@@ -1846,6 +1846,13 @@
     end
   end
 
+  context 'Tuple Sorter' do
+    it 'sorts data frames with nils' do
+      df = Daru::DataFrame.from_csv 'spec/fixtures/duplicates.csv'
+      expect { df.group_by(df.vectors.map(&:to_s)) }.to_not raise_error(ArgumentError)
+    end
+  end
```

Comment:
It should be where all other `group_by` specs are, as a separate `context`, named like `when there are nil in dataframe` or something like this.

And I don't think that separate fixture is necessary for testing this case, on-the-fly dataframe creation would be more visible.